const ExceptionMessage = {
  UNKNOWN_ERROR: 'Unknown error occurred.',
  EMAIL_ALREADY_EXISTS: 'Email is already taken.',
  INCORRECT_EMAIL: 'Incorrect email.',
  INVALID_TOKEN: 'Token is invalid.',
  PASSWORDS_NOT_MATCH: 'Passwords do not match.',
  USER_NOT_FOUND_BY_EMAIL: 'User was not found by email',
  USERNAME_ALREADY_EXISTS: 'Username is already taken.'
};

export { ExceptionMessage };
